src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_UsingStatement.cs

```diff
@@ -464,24 +465,41 @@ private BoundBlock MakeExpressionUsingStatement(BoundUsingStatement node, BoundB
         /// <summary>
         /// Synthesize a call `expression.Method()`, but with some extra smarts to handle extension methods, and to fill-in optional and params parameters.
         /// </summary>
-        private BoundExpression MakeCallWithNoExplicitArgument(SyntaxNode syntax, BoundExpression expression, MethodSymbol method)
+        private BoundExpression MakeCallWithNoExplicitArgument(Binder binder, SyntaxNode syntax, BoundExpression expression, MethodSymbol method)
         {
             var receiver = method.IsExtensionMethod ? null : expression;
 
-            var args = method.IsExtensionMethod
-                ? ImmutableArray.Create(expression)
-                : ImmutableArray<BoundExpression>.Empty;
+            var argsBuilder = ArrayBuilder<BoundExpression>.GetInstance();
+            if (method.IsExtensionMethod)
+            {
+                argsBuilder.Add(expression);
+            }
 
             Debug.Assert(!method.IsExtensionMethod || method.ParameterRefKinds.IsDefaultOrEmpty || method.ParameterRefKinds[0] != RefKind.Ref);
 
+            var expanded = method.HasParamsParameter();
+
+            var argsToParams = default(ImmutableArray<int>);
+            var refKindsBuilder = ArrayBuilder<RefKind>.GetInstance();
```

Comment:
It occurs to me that we could now pass `null` here and still get the behavior we want. Perhaps will address in a follow-up.